content/workload-reduction-toolkit/evaluate-workload-measures/update-your-workload-action-plan.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Update your workload action plan
+layout: /markdown_resource.html.erb
+colour: green
+---
+
+# Update your workload action plan
+
+<strong class="govuk-tag">Template</strong>
+
+## School details
```

Comment:
This section is missing an {inset-text} - it's missing the blue vertical line:

<img width="581" alt="Screenshot 2024-03-07 at 14 55 26" src="https://github.com/DFE-Digital/improve-school-workload-and-wellbeing/assets/18436946/70e4294c-2ba1-4174-85fd-e8416751eed1">